sql/core/src/test/resources/sql-tests/inputs/subquery/scalar-subquery/scalar-subquery-select.sql

```diff
@@ -1,5 +1,6 @@
 -- A test suite for scalar subquery in SELECT clause
 
+--ONLY_IF spark
```

Comment:
if most golden files are Spark only, shall we use opt-in approach and only run golden file with pgsql if it has a special marker?